stable/jenkins/master-image/Dockerfile

```diff
@@ -1,4 +1,4 @@
-FROM jenkins:2.7.4
+FROM jenkins:2.19.1
```

Comment:
There was another release  ,which  is `2.19.2` 
https://jenkins.io/changelog-stable/